includes/comment_controller.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+session_start();
+
+include "database_connection.php";
+include "../classes/CommentMethod.php";
+include "../classes/Post.php";
```

Comment:
I found out we don't need to include "../classes/Post.php" here so you can remove it